lib/install-theme.js

```diff
@@ -20,9 +31,18 @@ function installTheme(theme) {
   childProcess.execSync(installCommand, { stdio: 'inherit' });
 }
 
+/**
+ * Command handler for installing an @edx/brand package.
+ */
 async function themeCommand() {
-  const answers = await promptUserForTheme();
-  installTheme(answers.theme);
+  // Check if the user passed a theme parameter as a command-line argument
+  if (process.argv.length === 4) {
+    const providedTheme = process.argv[3];
+    installTheme(providedTheme);
```

Comment:
I decided to add it in this PR since it is small and I could use newly created `help` command functionality to describe it straight away.